djweb/dj_comp_hist/views.py

```diff
@@ -179,13 +179,22 @@ def browse(request):
     return render(request, 'browse.jinja2')
 
 def generate_search_facets(doc_objs):
+    """
+    Creates a dictionary with dictionaries of each facet of the given parameter. Each of the facet dictionaries contain
+    keys that are the top 10 most common instances of that given facet and values that signify the number of
+    occurrences of that specific instance.
+
+    :param doc_objs:
+    :return: dict_facets
+    """
     from collections import Counter
     counter_authors=Counter()
     counter_dates=Counter()
     counter_author_organizations=Counter()
     counter_recipients=Counter()
     counter_recipient_organizations=Counter()
-    counter_cc=Counter()
+    counter_cceds=Counter()
+    counter_cced_organizations=Counter()
     for doc in doc_objs.all():
         counter_dates[doc.date.year]=+1
```

Comment:
This error was thrown: 

~\Documents\GitHub\computation_hist\djweb\dj_comp_hist\views.py in generate_search_facets(doc_objs)
    213     counter_cced_organizations=Counter()
    214     for doc in doc_objs.all():
--> 215         counter_dates[doc.date.year]=+1
    216         for author in doc.author_person.all():
    217             counter_authors[author.fullname]=+1

AttributeError: 'NoneType' object has no attribute 'year'

There's no object called year; you'll have to find a different way to parse it.

Reply:
Hey Carol, that's super weird because I tested the code using embed and the advanced search. I was wondering what you typed in the shell? So, I could fix it.

Reply:
Some dates are None. I guess that makes sense (they are undated). 
You can just catch the error:
```
# Some dates are None -> skip those documents
if doc.date:
    counter_dates[doc.date.year] += 1
```